src/mscorlib/shared/System/Char.cs

```diff
@@ -26,7 +26,7 @@ public struct Char : IComparable, IComparable<Char>, IEquatable<Char>, IConverti
         //
         // Member Variables
         //
-        private char _value;
+        private char m_value; // Do not rename (binary serialization)
```

Comment:
The primitives shouldn't need renames since the serialization format special-cases them. The other changes should still be necessary.